### DISCUSSION TITLE
Issues when using 3270

### DISCUSSION THREAD

**Discussion by lbdyck** [zopencommunity/meta#195] [General]
See these screen captures with annotations

![image](https://user-images.githubusercontent.com/42328411/219689658-19416b1d-3c88-4756-82b5-2102b5935cbe.png)

**Comment by lbdyck** [comment]
Found the issue here - so do I need ncurses?

![image](https://user-images.githubusercontent.com/42328411/219691258-0b92b3f9-6e15-4237-9f2e-211935a562e1.png)

**Reply by IgorTodorovskiIBM** [reply]
Thanks @lbdyck , would it make sense to guard it like this?
```
if [ ! "${_BPX_TERMPATH-x}" = "OMVS" ]
  export TERM=xterm
 ...
fi
```

**Reply by lbdyck** [reply]
imho it would

**Reply by lbdyck** [reply]
I assume the test if it is not equal to OMVS then do the export

**Reply by IgorTodorovskiIBM** [reply]
Yep

**Reply by lbdyck** [reply]
I tried copy and paste and got this 

/.env 29: .: FSUM7332 syntax error: got fi, expecting then

**Comment by lbdyck** [comment]
This worked
![image](https://user-images.githubusercontent.com/42328411/219696241-696fe2cc-8f9f-4ec0-832f-12dabde7b07b.png)

**Reply by IgorTodorovskiIBM** [reply]
oh yes, forgot the "; then" :) Thanks

**Comment by lbdyck** [comment]
Now I'm getting this on my work z/OS: 
![image](https://user-images.githubusercontent.com/42328411/219697547-a9b7a86d-4561-4af1-bb72-fc924270d850.png)

Comparing my .bootenv between work and the CBT system - I had removed different things.

this is what I have on the cbt that has no problems `for p in meta curl git less perl bash diffutils findutils tar gzip bzip2 ncurses ; do  `
and this on my work system: `for p in meta curl git less perl bash ncurses ; do `

**Reply by IgorTodorovskiIBM** [reply]
A couple of questions:
* Did you pull in the latest ncurses? That changes TERMINFO to ASCII_TERMINFO (you'll also need to pull in less, git, etc if you did this)
* This happens after the above change? If TERM is set to dumb or some other value, does the less warning go away? Maybe we need to set TERM to legit value for OMVS

**Reply by lbdyck** [reply]
1. yes - it was a clean zopen-setup
2. yes

**Reply by lbdyck** [reply]
Test using a tn3270 and you should see it - unless it is just me :)

**Reply by lbdyck** [reply]
btw. my work system is z/OS 2.4 and the cbt system is z/OS 2.5

**Comment by lbdyck** [comment]
using the same .bootenv i still get the above error

**Comment by lbdyck** [comment]
Here is a word doc with the 2 environments - my work (bad) and cbt (good) in case you can see anything

[envs.docx](https://github.com/ZOSOpenTools/meta/files/10768450/envs.docx)

**Comment by lbdyck** [comment]
Note: this problem does NOT occur accessing z/OS OMVS via ssh

**Reply by IgorTodorovskiIBM** [reply]
<img width="331" alt="Screenshot 2023-02-17 at 10 56 03 AM" src="https://user-images.githubusercontent.com/39890068/219702486-cd7be0d7-b21b-460a-a1e2-3d6e92cbd25d.png">
Hey Lionel, I'm not able to see the env content

**Reply by lbdyck** [reply]
Open the word doc and then click on the attachments and you can save them and view them in notepad or other text viewer.

**Comment by lbdyck** [comment]
I found the solution - see this:
![image](https://user-images.githubusercontent.com/42328411/219866180-7530f223-eb1d-4b7f-87d1-56405ad855e8.png)

**Comment by MikeFultonDev** [comment]
@IgorTodorovskiIBM i think you are rolling out a fix for this - can you point @lbdyck to the fix when it's available and then @lbdyck you can close this off?

**Comment by MikeFultonDev** [comment]
This is now working I believe. @lbdyck please open an issue if not.
